pkg/geo/wkt/wkt.y

```diff
@@ -200,6 +200,11 @@ geometry:
 		if !ok {
 			return 1
 		}
+		err := $1.(*geom.GeometryCollection).SetLayout(wktlex.(*wktLex).curLayout())
+		if err != nil {
+			wktlex.(*wktLex).setError(err)
```

Comment:
i think we can only do this for 3D/M:

```

otan=# select 'geometrycollection ( point z empty )'::geometry;
                                   geometry
------------------------------------------------------------------------------
 0107000080010000000101000080000000000000F87F000000000000F87F000000000000F87F
(1 row)
```

Reply:
PostGIS accepts this, it parses as a `GEOMETRYCOLLECTION Z (POINT Z EMPTY)`.